.github/workflows/ci_apps_hightable_demo.yml

```diff
@@ -5,9 +5,9 @@ on:
       - 'apps/hightable-demo/**'
       - '.github/workflows/_common_jobs.yml'
       - '.github/workflows/ci_apps_hightable_demo.yml'
-      - 'shared.eslint.config.js'
+      - 'eslint.config.mjs'
```

Comment:
I think if we had `"type": "module"` in the package.json we could have left this as `.js`. (not a big deal)

Reply:
ups, right

Reply:
for the record, I changed because it was ignored otherwise in vscode, not sure why. Anyway, reverting and I'll look at the reason at another moment.

Reply:
OK, here is the reason:

https://github.com/hyparam/hyperparam-cli/actions/runs/12323081475/job/34397996892#step:4:17

> SyntaxError: Named export 'sharedJsRules' not found. The requested module '../../eslint.config.js' is a CommonJS module, which may not support all module.exports as named exports.